### PR TITLE
Export TOIL_TORQUE_ARGS

### PR DESCRIPTION
**Pull request by alaindomissy**
Environmental variable allowing passing of TORQUE scheduler specific parameters
https://github.com/BD2KGenomics/toil/issues/1658

### CODE REVIEW

File: src/toil/batchSystems/torque.py [hunk 1]
```diff
@@ -121,6 +121,15 @@ def prepareQsub(self, cpu, mem, jobID):
             if cpu is not None and math.ceil(cpu) > 1:
                 qsubline.extend(['-l ncpus=' + str(int(math.ceil(cpu)))])
 
+            # "Native extensions" for TORQUE (see DRMAA or SAGA)
+            nativeConfig = os.getenv('TOIL_TORQUE_ARGS')
+            if nativeConfig is not None:
+                logger.debug("Native TORQUE options appended to qsub from TOIL_TORQUE_RESOURCES env. variable: {}".format(nativeConfig))
+                #if ("--mem" in nativeConfig) or ("--cpus-per-task" in nativeConfig):
+                if ("mem=" in nativeConfig) or ("nodes=" in nativeConfig) or ("ppn=" in nativeConfig):
+                    raise ValueError("Some resource arguments are incompatible: {}".format(nativeConfig))
```
Comment (jessebrennan) [inline src/toil/batchSystems/torque.py]: Error message would be even more helpful if it listed the arguments that are not aloud.

File: src/toil/batchSystems/torque.py [hunk 2]
```diff
@@ -121,6 +121,15 @@ def prepareQsub(self, cpu, mem, jobID):
             if cpu is not None and math.ceil(cpu) > 1:
                 qsubline.extend(['-l ncpus=' + str(int(math.ceil(cpu)))])
 
+            # "Native extensions" for TORQUE (see DRMAA or SAGA)
+            nativeConfig = os.getenv('TOIL_TORQUE_ARGS')
+            if nativeConfig is not None:
+                logger.debug("Native TORQUE options appended to qsub from TOIL_TORQUE_RESOURCES env. variable: {}".format(nativeConfig))
+                #if ("--mem" in nativeConfig) or ("--cpus-per-task" in nativeConfig):
```
Comment (jessebrennan) [inline src/toil/batchSystems/torque.py]: Probably should delete this line, right?

File: src/toil/batchSystems/torque.py (hunk 1 above)
Comment (alaindomissy) [inline src/toil/batchSystems/torque.py]: true

**Comment by ejacox** [comment]
Also, please add the environment variable along with a description to the the table in here:
https://github.com/BD2KGenomics/toil/blob/master/docs/running/running.rst

**Comment by ejacox** [comment]
Jenkins, OK to test.

**Comment by ejacox** [comment]
Jenkins, test this please

**Comment by ejacox** [comment]
Jenkins, test this please.

**Comment by ejacox** [comment]
OK to test.

**Comment by ejacox** [comment]
restest this please

**Comment by ejacox** [comment]
Jenkins, add to whitelist

**Comment by ejacox** [comment]
Jenkins, restest this please

**Comment by ejacox** [comment]
Jenkins, ok to test

**Pull request merged**